pithos/application.py

```diff
@@ -85,26 +87,55 @@ def do_command_line(self, command_line):
         # http://stackoverflow.com/questions/1943747/python-logging-before-you-run-logging-basicconfig
         logging.root.handlers = []
 
+        # Show the Pithos log since last reboot and exit
+        if options.contains('last-logs'):
+            try:
+                from systemd import journal
+                from os.path import basename
+            except ImportError:
+                self._print(command_line, _('Systemd Python module not found'))
+                return 0
```

Comment:
I don't understand the question, On an error processes just exit with a non-zero number, any will do.